### PR TITLE
Remove flex from Stepwise list items and add inline media styles

### PR DESCRIPTION
**Pull request by jivey**
There are two changes here [to solve a problem with stepwise list items](https://github.com/openstax/cnx-recipes/issues/4939). A quick overview:

1. **Stepwise list items**. Remove flex from these to prevent the overlap render cases below. This seems like a safe change; when looking through the [Sifter](https://openstax.github.io/sifter/#v=1&q=.stepwise&sourceFormat=xhtml) results, I didn't see anything that required the flex style. I could be missing some context for that rule though.

| Before | After |
---|---
| image | image |
| image | image |

2. **Inline media**. The media inline change is related to the first, when removing the flex, the stepwise media content drops to the next line, and non-stepwise media had some existing alignment issues, so I'm trying to resolve both of these with the media inline rule. 

| Stepwise Before | Stepwise After |
---|---
| image | image |

| Non-Stepwise Before | Non-Stepwise After |
---|---
| image | image |

### CODE REVIEW

File: styles/designs/webview/parts/_lists-components.scss [hunk 1]
```diff
@@ -100,9 +100,7 @@ $StepwiseList__Container: (
 $StepwiseList__Item: (
     _name: "StepwiseListItem",
     _subselector: ' > li',
-    _properties: (
-      display: flex,
-    )
+    _properties: ()
 );
```
Comment (wilkus27) [inline styles/designs/webview/parts/_lists-components.scss:104]: When component doesn't have any property we use `empty_wrapper` function.
E.g. `$ChapterOutline: empty_wrapper(ChapterOutline, ' .os-chapter-outline');`

**Review by wilkus27 — CHANGES_REQUESTED** [review]
It looks good :relaxed: I left one comment about one of the component. Beside that, could you change base branch? In `ce-styles` we are merging first to milestone branch. Here is a PR for current one: https://github.com/openstax/ce-styles/pull/144 :wink:

**Comment by jivey** [comment]
@wilkus27 Thank you for the review! Updated to this: `$StepwiseList__Item: empty_wrapper(StepwiseListItem, ' > li');`

**Review by wilkus27 — APPROVED** [review]
Approved! :wink:

**Pull request merged**